app/uk/gov/hmrc/incometaxsoftwarechoicesfrontend/views/ProductDetailsPage.scala.html

```diff
@@ -56,6 +51,12 @@
 
 @mainTemplate(title = messages("search-software.title"), breadcrumbs = Some(pageBreadcrumbs)) {
 
-  <p> Product details for software vendor "@{softwareVendor}" </p>
+  <dl class="govuk-summary-list contact-details">
```

Comment:
Whilst I understand why this was done, I don't think this is using definition lists correctly and might cause confusion for users with assistive technology.
Instead we should probably just use page layout classes to do this. Maybe one third columns for each part?